app/javascript/react/components/CollectionShow.js

```diff
@@ -16,12 +16,15 @@ const CollectionShow = (props) => {
     )
   })
   return (
-    <div>
-      <img src={props.piece_photo}/> <br />
+    <div className="grid-x grid-margin-x callout">
+      <div className="collection-piece-photo">
+        <img  src={props.piece_photo}/> <br />
```

Comment:
we should opt for CSS to handle styling like line breaks - that way all styling is managed in the same place (CSS) rather than some in CSS, and some "inline"